README.rst

```diff
@@ -52,7 +52,7 @@ Using pip::
 
     pip install torchtext
 
-**Note** LTS versions are distributed through a different channel than the other versioned releases. 
+**Note** LTS versions are distributed through a different channel than the other versioned releases.
```

Comment:
NIT: Can we remove this change as it's not related to this PR?

Reply:
Done.